account_invoice_rounding/account.py

```diff
@@ -57,28 +57,11 @@ def _swedish_add_invoice_line(self, cr, uid, invoice,
         return {'amount_total': rounded_total,
                 'amount_untaxed': amount_untaxed}
 
-    @staticmethod
-    def _all_invoice_tax_line_computed(invoice):
-        """ Check if all taxes have been computed on invoice lines
-        :return boolean True if all tax were computed
-        """
-        tax_ids = set()
-        for line in invoice.invoice_line:
-            # invoice_line_tax_id is a many2many if you wonder about it
-            for tax in line.invoice_line_tax_id:
-                if not tax.price_include:
-                    tax_ids.add(tax.id)
-        computed_tax_ids = [tax.id for tax in invoice.tax_line]
-        return len(tax_ids) == len(computed_tax_ids)
-
     def _swedish_round_globally(self, cr, uid, invoice,
```

Comment:
Why Swedish? Is there any special rounding that does not apply anywhere else?
If possible I would rename to _round_globally()